### PR TITLE
Fix koala position in emoji lists

### PR DESCRIPTION
**Pull request by nathandud**
Fixes issue #86. Emoji Search is broken for ~60 animals due to a mismatch between emojis and their descriptions in the hard-coded emoji lists.

### CODE REVIEW

**Review by Fintasys — COMMENTED** [review]
Thanks for your PR and finding/solving this issue !
If you don't mind I would prefer to change the position of the description rather than the koala's face position because the animal heads suppose to be all together.

**Comment by nathandud** [comment]
@Fintasys I have updated the PR accordingly - the description is simply "Koala" in the original source. Do you want that updated to "Koala Face" to match the others?

File: lib/src/emoji_lists.dart [hunk 1]
```diff
@@ -739,6 +739,7 @@ final Map<String, String> animals = Map.fromIterables([
   'Fox Face',
   'Bear Face',
   'Panda Face',
+  'Koala',
```
Comment (Fintasys) [inline lib/src/emoji_lists.dart:742]: Ah yeah, "Koala Face" seems to be the most correct term here 🙇 Please change! 🙏

Comment (nathandud) [reply lib/src/emoji_lists.dart]: ✅

**Pull request merged**